pallets/loans/src/entities/loans.rs

```diff
@@ -344,25 +344,25 @@ impl<T: Config> ActiveLoan<T> {
 		&self,
 		mut amount: RepaidPricingAmount<T>,
 	) -> Result<RepaidPricingAmount<T>, DispatchError> {
-		let (interest_accrued, max_repay_principal) = match &self.pricing {
+		let (outstanding_interest, max_repay_principal) = match &self.pricing {
```

Comment:
Tiny detail but I think it would make sense to swap the tuple order here, as in all other cases principal comes before interest.

Reply:
Totally agree!